backend/game.js

```diff
@@ -349,7 +413,7 @@ module.exports = class Game extends Room {
         this.meta();
       return;
     }
-
+    
```

Comment:
I guess these are useless

Reply:
Hmm guess the linter isn't configured to flag extra whitespace. Fixed https://github.com/dr4fters/dr4ft/pull/990/commits/629e5779e1ab5dccd35ed5e7b9c54236e26b2fd2